### PR TITLE
Coordination Meeting 224, 2022-03-03

### PR DESCRIPTION
**Pull request by graeme-a-stewart**
There is a false positive link checker failure that might come up to `./plan-of-work-2021.html`, but this is just because the link checker has issues with internal relative links (#1046).

### CODE REVIEW

File: organization/_posts/2022/2022-03-03-coordination.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+- Kick-off meeting arranged for 25 March:
+  - <https://indico.cern.ch/event/1132360/>
+
+---
+
+## AOB
+
+### Calendar
+
+Q. How do we setup the HSF calendar?
```
Comment (amete) [inline organization/_posts/2022/2022-03-03-coordination.md:147]:
```suggestion
Q. How do we set up the HSF calendar?
```

File: organization/_posts/2022/2022-03-03-coordination.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+
+Draft presentation will be ready on Monday 7.
+
+### Workshops in 2022
+
+#### Analysis Ecosystems Workshop
+
+23-25 May: <https://indico.cern.ch/e/aew2>
+
+**Registration is opened!** At the moment we have limited this to 80 in-person places.
```
Comment (amete) [inline organization/_posts/2022/2022-03-03-coordination.md:24]:
```suggestion
**Registration is open!** At the moment we have limited this to 80 in-person places.
```

**Review by amete — APPROVED** [review]
I have two minor suggestions but apart from those all looks good to me, thanks a lot @graeme-a-stewart.

**Comment by graeme-a-stewart** [comment]
Thank you @amete !

**Pull request merged**